### DISCUSSION TITLE
Welcome to lambdanative Discussions!

### DISCUSSION THREAD

**Discussion by mgorges** [part-cw/lambdanative#420] [General]
## 👋 Welcome!
  We’re using Discussions in place of the soon-to-be discontinued [Spectrum chat](https://spectrum.chat/lambdanative) as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cidne** [comment]
Hello.

Love what you all are doing--I'm just going to look around for now.  

Thanks!
